src/test/scala/uk/gov/hmrc/test/ui/pages/base/DriverHelper.scala

```diff
@@ -112,9 +111,8 @@ trait DriverHelper {
   }
 
   def fillTextBoxById(elementId: String, text: String): Unit = {
-    val element = waitForId(elementId)
-    element.clear()
-    element.sendKeys(text)
+    waitForId(elementId).clear()
+    waitForId(elementId).sendKeys(text)
```

Comment:
The former implementation was better. Why do you need to wait 2 times for the same element?

Reply:
elementId is a string and without just providing element.sendKeys(text) returns an error because it is expecting webelement